js_scripts/UserScripts.js

```diff
@@ -0,0 +1,124 @@
+function findUserName(id, model){
+    model.findById(id, 'Profile',function(err, member){
+        if(err){
+            console.log(err);
+        }else{
+            console.log(member.Profile.name);
+        }
+    })
+}
+function findUserEmail(id, model){
+    model.findById(id, 'Profile',function(err, member){
+        if(err){
+            console.log(err);
+        }else{
+            console.log(member.Profile.email);
+        }
+    })
+}
+
+function findUserPassword(id, model){
+    model.findById(id, 'Profile',function(err, member){
+        if(err){
+            console.log(err);
+        }else{
+            console.log(member.Profile.password);
+        }
+    })
+}
+
+
+async function findUserId(email, model){
+    model.findOne({"Profile.email": email}, 'Profile').then(
```

Comment:
Had to turn this function into an async function, otherwise it would always return undefined

Reply:
I am not sure why you got undefined. I was testing it on my side and it worked just fine.